share/transform/chain/utils.py

```diff
@@ -36,3 +41,34 @@ def format_address(address1='', address2='', city='', state_or_province='', post
         return '{}, {}'.format(city, state_or_province)
 
     return address1
+
+
+def force_text(data):
```

Comment:
@sheriefvt, this is so much better 🎉 . Can you add some tests for `force_text`? That way we can be sure that it is getting everything.